lib/sqlalchemy/orm/base.py

```diff
@@ -319,11 +319,7 @@ def object_state(instance):
 def _inspect_mapped_object(instance):
     try:
         return instance_state(instance)
-        # TODO: whats the py-2/3 syntax to catch two
-        # different kinds of exceptions at once ?
-    except exc.UnmappedClassError:
-        return None
-    except exc.NO_STATE:
+    except (exc.UnmappedClassError, exc.NO_STATE):
```

Comment:
NO_STATE is a tuple itself, so we cannot do this as evidenced by the test failing
https://github.com/sqlalchemy/sqlalchemy/blob/ea87d39d7a9926dc1c6bf3d70e8faf8575769cb0/lib/sqlalchemy/orm/exc.py#L15

Reply:
That's a pretty obvious and early fail indeed. I think I ran the test suite against `master` before I suggested this. I've updated the PR with a small helper function (and one additional site where a similar pattern is used), but it's less convincingly an improvement.